provisioners/roles/ace/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+# tasks file for ace
+
+- name: determine whether ace-builds already exists and dont replace
```

Comment:
This may not be necessary.  The `git` module will check to see if the repo exists already, and we want it to be pegged at a specific version anyway.  I think you can take this one out.